src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -108,7 +108,7 @@ internal void SetLocalEndpointFromResource(Resource resource)
             }
 
             string serviceName = null;
-            Dictionary<string, object> tags = null;
+            Dictionary<string, object> tags = new Dictionary<string, object>();
```

Comment:
if we won't use, should we remove this?

Reply:
I actually wasn't sure about this entire static method as a whole (SetLocalEndpointFromResource), it seemed like the foreach loop iterating over the attributes was written for something else, but not ultimately used? I know that the loop is necessary to check the string keys for serviceName though.

Reply:
It has gone through a few iterations. We also used to pull library name + version from the resource (in the loop) but now we get that from ActivitySource. It is probably safe to remove the `tags` local and just pass `null` to the ctor eg:
```csharp
this.LocalEndpoint = new ZipkinEndpoint(
   serviceName,
   ipv4,
   ipv6,
   port: null,
   tags: null);
```
You could also switch the `continue` to break out of the loop since we are only looking for one thing:
```csharp
if (key == ResourceSemanticConventions.AttributeServiceName)
{
   serviceName = label.Value as string;
   break;
}
```